### PR TITLE
Italian translation of performance > resource-prioritization

### PR DESCRIPTION
**Pull request by lucab85**
Italian translation of [/performance/resource-prioritization](https://developers.google.com/web/fundamentals/performance/resource-prioritization).

Please review by @carlaofarrell

### CODE REVIEW

**Comment by agektmr** [comment]
Thanks @lucab85 for translating this article! Once review is done, I'll pull this change.
It would be great if you could [try using GitLocalize](https://developers.google.com/web/resources/translations?hl=en) next time.

**Comment by lucab85** [comment]
Thnak you @agektmr.
I've started a Review in GitLocalize:  https://gitlocalize.com/repo/107/it/review/376

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+recuperare il contenuto, è lo streaming multimediale da un'origine diversa.
+
+A seconda di come la tua pagina gestisce il contenuto in streaming, potresti
+voler aspettare che gli script siano caricati e pronti per elaborare il flusso.
+Preconnect ti aiuta a ridurre i tempi di attesa per un singolo roundtrip una
+volta che sei pronto per iniziare il recupero.
+
+## Prefetch
+
+`<link rel="prefetch">` è in qualche modo diverso da `<link rel="preload">` e
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: `<link rel="prefetch">` è piuttosto diverso da `<link rel="preload">` e

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+`<link rel="preconnect">`, nel senso che non tenta di far succedere qualcosa di
+critico più velocemente; invece, cerca di far succedere qualcosa di non critico
+prima, se possibile.
+
+Lo fa informando il browser di una risorsa che dovrebbe essere necessaria come
+parte di una futura navigazione o interazione dell'utente, ad esempio,
+qualcosa che *potrebbe* essere necessario in seguito, se l'utente intraprende
+l'azione che siamo aspettando. Queste risorse vengono recuperate con la priorità
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: l'azione ci aspettiamo. Queste risorse vengono recuperate con la priorità

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+**più bassa** in Chrome, quando viene caricata la pagina corrente e c'è
+larghezza di banda disponibile.
+
+Ciò significa che `prefetch` è più adatto a prevedere ciò che l'utente potrebbe
+fare in seguito e prepararsi, come recuperare la prima pagina dei dettagli del
+prodotto in un elenco di risultati o recuperare la pagina successiva in
+contenuto impaginato.
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: un contenuto impaginato.

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+      <head>
+        <link rel="prefetch" href="optional.css">
+        <link rel="stylesheet" href="optional.css">
+      </head>
+      <body>
+        Hello!
+      </body>
+    </html>
+
+Tuttavia, questo farà sì che il tuo foglio di stile venga prelevato due volte
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: Tuttavia, questo farà sì che il tuo foglio di stile venga recuperato due volte

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+    <img src="images/res-prio-prefetch.png"
+         alt="Uno screenshot di Chrome Developer Tools che mostra optional.css che
+              viene recuperato due volte">
+  </div>
+</figure>
+
+Il doppio recupero può essere negativo per gli utenti. In questo caso, non solo
+avrebbero dovuto attendere il CSS che bloccava il rendering, ma avrebbero anche
+potenzialmente perso larghezza di banda scaricando il file due volte. Ricorda
+che la loro larghezza di banda potrebbe essere contabilizzata. Assicurati di
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: che la loro larghezza di banda potrebbe essere in base al consumo. Assicurati di

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+in anticipo il browser in modo dichiarativo riguardo a risorse e connessioni ed
+ottimizzare l'ordine delle attività in relazione alle reali necessità.
+
+Esistono numerosi altri strumenti e tecniche che è possibile utilizzare per
+modificare la priorità ed i tempi di caricamento delle risorse. Assicurati di
+leggere
+[HTTP/2 server push](/web/fundamentals/performance/http2/#server_push);
+[utilizzare `IntersectionObserver` per carimanto lazy di immagini ed altri media](/web/updates/2016/04/intersectionobserver);
+[evitare CSS che bloccano il render](/web/fundamentals/performance/critical-rendering-path/render-blocking-css)
+con media queries e librerie come
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: con media query e librerie come

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 7]
```diff
@@ -0,0 +1,369 @@
...
+`<link rel="preload">`, `<link rel="preconnect">` e `<link rel="prefetch">`
+(ed il bonus `<link rel="dns-prefetch">`) sono ottimi strumenti per informare
+in anticipo il browser in modo dichiarativo riguardo a risorse e connessioni ed
+ottimizzare l'ordine delle attività in relazione alle reali necessità.
+
+Esistono numerosi altri strumenti e tecniche che è possibile utilizzare per
+modificare la priorità ed i tempi di caricamento delle risorse. Assicurati di
+leggere
+[HTTP/2 server push](/web/fundamentals/performance/http2/#server_push);
+[utilizzare `IntersectionObserver` per carimanto lazy di immagini ed altri media](/web/updates/2016/04/intersectionobserver);
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/performance/resource-prioritization.md]: [utilizzare `IntersectionObserver` per il caricamento lazy di immagini ed altri media](/web/updates/2016/04/intersectionobserver);

**Comment by carlaofarrell** [comment]
Hi Luca,

 

The review of the article is now complete!

**Comment by agektmr** [comment]
@carlaofarrell @lucab85  is this good to be merged?

**Comment by carlaofarrell** [comment]
Yes, good to me!

**Comment by lucab85** [comment]
Good for me too!

**Comment by agektmr** [comment]
@lucab85 have you sent a PR to [add yourself as a contributor](https://developers.google.com/web/resources/translations#credit) yet? If not, please do soon.

Regarding the moderator, I can add you as one for now. How does that sound? Please ask someone else for reviews, but you should be able to send a PR yourself.

**Comment by agektmr** [comment]
Actually, adding the contributor code in this branch would be easier for me.

**Comment by lucab85** [comment]
Thank you for the remind but I'm already in the list as [contributor](https://developers.google.com/web/resources/contributors/#luca-berton).
I'll be glad to be a moderator for the Italian translations.
@carlaofarrell would be a great reviewers, isn't it @petele ?

**Comment by agektmr** [comment]
OK, interesting. I saw an error `{% include "web/_shared/contributors/lucaberton.html" %}` part in local environment but it actually works in production. I kind of see why it is. but anyway, I can merge this now then.

**Pull request merged**